docs/components/user/user-button.mdx

```diff
@@ -40,18 +40,20 @@ In this example, `<UserButton />` is mounted inside a header component, which is
               <SignInButton />
             </SignedOut>
           </header>
-      );
+        );
       }
 
-      function MyApp({ pageProps }) {
+      export default function RootLayout({ children }: { children: React.ReactNode }) {
         return (
-          <ClerkProvider {...pageProps}>
-            <Header />
-          </ClerkProvider>
+          <html lang="en">
+            <ClerkProvider>
+              <Header />
+            </ClerkProvider>
+          </html>
         );
       }
 
-      export default MyApp;
+      export default RootLayout;
```

Comment:
The layout is already exported above.

```suggestion
```

Reply:
I forgot to delete this line when I moved the export 😭